### PR TITLE
fix: tensorflow 2.3 cpu

### LINKED ISSUE #64: fix: tensorflow 2.3 layer with only GPU
**Environment:** both

## Actual Behaviour
Currently the CPU version of the image is using the same version of the tensorflow as the GPU

## Expected behaviour
It should use different packages for each, as it is done in the tensorflow 1.15

### PR DESCRIPTION
**Pull request by victoriavilasb**
Closes #64

### CODE REVIEW

File: data-science/jupyterlab-tensorflow-2/requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-tensorflow==2.3.*
\ No newline at end of file
+tensorflow-cpu
```
Comment (portellaa) [inline data-science/jupyterlab-tensorflow-2/requirements.txt:1]: missing version

File: data-science/jupyterlab-tensorflow-2/requirements.txt [hunk 2]
```diff
@@ -1 +1 @@
-tensorflow==2.3.*
\ No newline at end of file
+tensorflow-cpu==2.3.1
```
Comment (rafaelcpalmeida) [inline data-science/jupyterlab-tensorflow-2/requirements.txt:1]: Why locking on 2.3.1 and not having a wildcard on the patch version?

**Comment by rafaelcpalmeida** [comment]
According to the [docs](https://www.tensorflow.org/install/gpu#older_versions_of_tensorflow), `tensorflow-gpu` and `tensorflow-cpu` separation is just for the 1.15 and lower versions. 🤔

File: data-science/jupyterlab-tensorflow-2/requirements.txt (hunk 1 above)
Comment (victoriavilasb) [reply data-science/jupyterlab-tensorflow-2/requirements.txt]: resolved

**Comment by portellaa** [comment]
> According to the [docs](https://www.tensorflow.org/install/gpu#older_versions_of_tensorflow), `tensorflow-gpu` and `tensorflow-cpu` separation is just for the 1.15 and lower versions. 🤔

no, in this case it's correct. 
- for v1: tensorflow is cpu bounded, so if you want gpu you have to use tensorflow-gpu
- for v2: tensorflow is gpu bounded, so if you want cpu you have to use tensorflow-cpu

**Review by portellaa — CHANGES_REQUESTED** [review]
this will remove the gpu version. we need to have both

**Comment by portellaa** [comment]
don't forget to link #64

**Review by portellaa — APPROVED** [review]

**Pull request merged**